plonky2x/core/src/frontend/vars/array.rs

```diff
@@ -175,6 +174,17 @@ impl<L: PlonkParameters<D>, const D: usize> CircuitBuilder<L, D> {
         V::from_variables_unsafe(&selected_vars)
     }
 
+    /// Given an `array` of variables, and a dynamic `index` start_idx, returns `array[start_idx..start_idx+sub_array_size]` as an `array`.
+    /// The `seed` is used to generate randomness for the proof.
+    /// The security of each challenge is log2(field_size) - log2(array_size), so the total security is log2(field_size) - log2(array_size) * num_loops.
```

Comment:
nit: should have extra parenthesis because of order of operations: 

(log2(field_size) - log2(array_size)) * num_loops.